.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 .DS_Store
+.idea
```

Comment:
Not familiar with this—is it something that can appear throughout the source tree, or normally just at the root?

Reply:
_(Thanks for calling this out. This is my test checkin and appreciate the extra scrutiny.)_

Webstorm (and all Jetbrains IDEs like IntelliJ/Kotlin) create a hidden directory `.idea/` where it stores all it's configuration files.

It can appear in various places depending on where a project is "opened".  For example, I open the frontend directory with Webstorm directory and a .idea directory is created there.

The `backend` directory's `.gitignore` already has it. In theory, the exclusion could have been added to the root `.gitignore` and both sub directories would have been covered.

If you prefer, I can add the `.idea` just to the `frontend`'s `.gitignore` 

Or, I can add a comment (which I probably should have done anyways)

Reply:
A comment is always appreciated (as is the detailed explanation!)—I don't see a reason to not have it in the root .gitignore (and I don't think there's a huge problem with having it in two places, though I'm a neat-freak about repositories, so if it were me I would probably delete it from frontend/.gitignore), I was just wondering if I should think of it as similar to `.eclipse` or similar to `.DS_Store`. Sounds more like the former—thanks!

And yes, I really do overuse parentheses like that all the time. Just in English, though!